components/pages/login/components/LoginForm/LoginForm.jsx

```diff
@@ -34,11 +34,11 @@ const LoginForm = () => {
 
   useEffect(() => {
     if (detailMessage) setSuccess(detailMessage);
-  }, [detailMessage]);
+  }, [detailMessage, setSuccess]);
```

Comment:
are u sure that we need to add function in array of dependence?
it does't have meaning there